lib/editor/components/ExceptionDateRange.js

```diff
@@ -140,7 +141,7 @@ export default class ExceptionDateRange extends Component<Props, State> {
             className='date-range'
             dateTime={range.endDate ? +moment(range.endDate) : +moment(range.startDate).add(1, 'days')}
             inputProps={{style: dateTimeProps}}
-            key={`date-${index}-end`}
+            key={uuidv4()}
             minDate={startMoment && startMoment.clone().startOf('day').add(2, 'days')} // See above, minDate subtracts an extra day.
             onChange={(millis) => this._onRangeUpdate(millis, false)}
             {...dateTimeProps}
```

Comment:
One thing we didn't catch in #828: if there is a validation error on a date range, the corresponding message should be shown.

Reply:
Ranges will now show an error as of https://github.com/ibi-group/datatools-ui/pull/843/commits/a83521ba6f5a2e66f8642193ca4a3c27efd71908